endpoint/response.go

```diff
@@ -75,20 +83,29 @@ func EncodePhonebook(data []*model.Phonebook) []*Phonebook {
 		}
 
 		encodeData := &Phonebook{
-			ID:           v.ID,
-			PhoneNumbers: phoneNumbers,
-			Description:  v.Description,
-			Name:         v.Name,
-			Address:      v.Address,
-			Latitude:     v.Latitude,
-			Longitude:    v.Longitude,
-			Status:       v.Status,
-			Category:     v.Category,
-			Distance:     v.Distance,
+			ID:            v.ID,
+			PhoneNumbers:  phoneNumbers,
+			Description:   v.Description,
+			Name:          v.Name,
+			Address:       v.Address,
+			CoverImageURL: fmt.Sprintf("%s/%s", cfg.AppStoragePublicURL, v.CoverImageURL),
+			Latitude:      v.Latitude,
+			Longitude:     v.Longitude,
+			Status:        v.Status,
+			StatusLabel:   GetStatusLabel[v.Status]["ina"],
+			Category:      v.Category,
+			Distance:      v.Distance,
 		}
 
 		result = append(result, encodeData)
 	}
 
 	return result
 }
+
+// GetStatusLabel ...
+var GetStatusLabel = map[int64]map[string]string{
+	-1: {"en": "status deleted", "ina": "Dihapus"},
```

Comment:
```suggestion
	-1: {"en": "Status Deleted", "ina": "Dihapus"},
```

I suggest to make it capital.

Reply:
![image](https://user-images.githubusercontent.com/29980674/124415861-ee16f880-dd7f-11eb-8322-1c9a0a9d2ef1.png)
![image](https://user-images.githubusercontent.com/29980674/124415886-fb33e780-dd7f-11eb-8287-32419fdcf099.png)
I think, on the exists contract, it use sentence case